aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js

```diff
@@ -0,0 +1,322 @@
+  },
+  {
+    name: 'Performance',
+    icon: <Optimize />,
+    layout: 'cards',
+    parent: 'Home',
+    children: ['Sessions', 'Logs', 'Diagnostics'],
+  },
+  {
+    name: 'Devices',
+    icon: <Servers />,
+    layout: 'cards',
+    parent: 'Home',
+    children: ['Server Cluster', 'Beacon Cluster'],
+  },
+  {
+    name: 'Users',
+    icon: <Group />,
+    layout: 'list',
+    parent: 'Home',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Sessions',
+    icon: <ListIcon />,
+    layout: 'list',
+    parent: 'Performance',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Logs',
+    icon: <UnorderedList />,
+    layout: 'list',
+    parent: 'Performance',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Remote Access',
+    icon: <Location />,
+    layout: 'panes',
+    parent: 'Home',
+  },
+  {
+    name: 'Security',
+    icon: <ShieldSecurity />,
+    layout: 'panes',
+    parent: 'Home',
+  },
+  {
+    name: 'Diagnostics',
+    icon: <Overview />,
+    layout: 'panes',
+    parent: 'Performance',
+  },
+  {
+    name: 'Server Cluster',
+    icon: <Cluster />,
+    layout: 'cards',
+    parent: 'Devices',
+    children: ['Server A', 'Server B'],
+  },
+  {
+    name: 'Beacon Cluster',
+    icon: <Cluster />,
+    layout: 'cards',
+    parent: 'Devices',
+    children: ['Beacon A', 'Beacon B', 'Beacon C', 'Beacon D'],
+  },
+  {
+    name: 'Server A',
+    icon: <Server />,
+    layout: 'panes',
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Server B',
+    icon: <Server />,
+    layout: 'panes',
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Beacon A',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon B',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon C',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon D',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+];
+
+const getPageDetails = pageName => pages.find(page => page.name === pageName);
+
+export const HubSpokeCardsExample = ({ mobile }) => {
+  const [currentPage, setCurrentPage] = React.useState(getPageDetails('Home'));
+  const [subPages, setSubPages] = React.useState();
+
+  React.useEffect(() => {
+    const pageDetails =
+      currentPage.children &&
+      currentPage.children.map(child =>
+        pages.find(page => page.name === child),
+      );
+    setSubPages(pageDetails);
+  }, [currentPage]);
+
+  const { children, layout, name, parent } = currentPage;
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <AppContainer>
+        <AppHeader currentPage={currentPage} setCurrentPage={setCurrentPage} />
+        {typeof parent !== 'undefined' && (
+          <SimpleButton onClick={() => setCurrentPage(getPageDetails(parent))}>
```

Comment:
Should we wrap this in a Box with align="start" so the button width doesn't span the whole screen?
Notice here on hover:
<img width="1231" alt="Screen Shot 2020-05-06 at 4 27 23 PM" src="https://user-images.githubusercontent.com/12522275/81238378-c3cc7e00-8fb6-11ea-83ff-9095d8875bf7.png">
Good news is that it appeared on Chrome, Safari, and Firefox

Reply:
Yes, we should. Could you do me a favor and try that in your local environment? When I did that earlier, I was having similar funny business to what I was showing you earlier -- particularly in pages which the tiles / lists overflow.

Reply:
not sure which issue you guys are referring to but in my local I did `alignSelf="start"` and didnt see any issues

Reply:
Nice! I will try that in the AM.